### DISCUSSION TITLE
[loco-translation-provider] Installation as require-dev only

### DISCUSSION THREAD

**Discussion by amigian74** [symfony/symfony#45201] [Q&A] [answered]
Hi.

I want to use the loco-translation-provider to retrieve my translataion directly, but I don't want this bundle in our production environment so I did install this package as require-dev. Unfortunately it doesn't work.

```
In TranslationProviderCollectionFactory.php line 55:

  The "loco" scheme is not supported.  
```

Can anybody help?

Thx in advance

**Comment by Broutard** [comment, marked as answer]
Enable the "loco" provider only in your dev environment by using `when@dev` in your `translation.yaml` config (for example).
